src/main/java/types/MigrationOptions.java

```diff
@@ -41,6 +41,29 @@ public interface MigrationOptions {
      */
     InheritableBoolean encrypted();
 
+    /**
+     * Specifies whether and how to use parallel migration connections.
+     *
+     * @author Milan Zamazal <mzamazal@redhat.com>
+     * @date 24 May 2022
+     * @status added
+     * @since 4.5.1
+     */
+    ParallelMigrations parallelMigrations();
```

Comment:
I think ParallelMigrations isn't the best name for an enum. Perhaps more suitable here would be ParallelMigrationsPolicy?